src/docfx/build/dependency/DependencyType.cs

```diff
@@ -10,6 +10,7 @@ public enum DependencyType
         UidInclusion, // uid reference with display property
         Inclusion, // token or codesnippet
         Overwrite, // overwrite markdown reference
-        TocInclusion, // toc reference toc
+        TocInclusion, // toc includes toc
+        TocFolderInclusion, // toc includes toc first item
```

Comment:
naming: what about `TocFolderReference`? Just to make it clear that it is not an inclusion.

Reply:
it's treated as inclusion in v2

Reply:
sorry, toc and toc relationships are missing in v2, so v3 generate it according to our own understanding.